src/modules/itslearning/actions/create-memberships.action.ts

```diff
@@ -0,0 +1,66 @@
+import { DomainError } from '../../../shared/error/domain.error.js';
+import { IMS_COMMON_SCHEMA, IMS_MEMBER_MAN_DATA_SCHEMA, IMS_MEMBER_MAN_MESS_SCHEMA } from '../schemas.js';
+import { IMSESRoleType } from '../types/role.enum.js';
+import { IMSESMassAction } from './base-mass-action.js';
+
+// Incomplete
```

Comment:
What do you mean by that?

Reply:
The real action can take more inputs than this, but I've limited it to the inputs we need. 
Therefore, the properties shown here are incomplete.

I've used this convention in the other actions as well